# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cache/nsCacheService.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d netwerk/cache/nsCacheService.cpp
--- a/netwerk/cache/nsCacheService.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/netwerk/cache/nsCacheService.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -357,7 +357,7 @@
     mKey.Append(key);
     mStoragePolicy = session->StoragePolicy();
     mListener = listener;
-    mEventTarget = GetCurrentThreadEventTarget();
+    mEventTarget = GetCurrentEventTarget();
     // We addref the listener here and release it in nsNotifyDoomListener
     // on the callers thread. If posting of nsNotifyDoomListener event fails
     // we leak the listener which is better than releasing it on a wrong
@@ -908,7 +908,7 @@
   if (!listener) return NS_OK;  // we're sync, we're done.
 
   // get the request's thread
-  (*request)->mEventTarget = GetCurrentThreadEventTarget();
+  (*request)->mEventTarget = GetCurrentEventTarget();
 
   return NS_OK;
 }